Cargo.toml

```diff
@@ -10,6 +10,8 @@ rayon = "^1.0"
 ispc = { version = "^1.0.4", optional = true }
 structopt = { version = "0.3.0", features = ["color"] }
 wide = "0.7.28"
+pulp = { version = "0.20.1", features = ["nightly"] }
```

Comment:
Does `pulp` require nightly? Part of my motivation here is to showcase libraries that _don't_ require nightly, i.e. where `std::simd` isn't an option.

Reply:
it doesn't. it only requires it for avx512 support (until it's stabilized)

i wanted it so the benchmark would be a fair fight

Reply:
the code should still work as-is without the nightly flag, it'll just be slower on avx512 machines

Reply:
Makes sense, thanks for the explanation.